### PR TITLE
Rename to f8a worker

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

**Comment by fridex** [comment]
These patches have cyclic dependencies so it will not be possible to make CI happy....

Related:
   * https://github.com/fabric8-analytics/fabric8-analytics-common/pull/16
   * https://github.com/fabric8-analytics/fabric8-analytics-worker/pull/14
   * https://github.com/fabric8-analytics/fabric8-analytics-server/pull/12
   * https://github.com/fabric8-analytics/fabric8-analytics-jobs/pull/16

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Project Fabric8-Analytics
 
-*Note on naming: The Fabric8-Analytics project has evolved from 2 different projects called "cucos" and "bayesian". We're currently in process of renaming the modules and updating documentation. Until that is completed, please consider "cucos" and "bayesian" to be synonyms of "Fabric8-Analytics".*
+*Note on naming: The Fabric8-Analytics project has evolved from 2 different projects called "f8a" and "bayesian". We're currently in process of renaming the modules and updating documentation. Until that is completed, please consider "f8a" and "bayesian" to be synonyms of "Fabric8-Analytics".*
```
Comment (jpopelka) [inline README.md]: Your renaming algorithm is too greedy here, but this note will be renamed eventually, so no problem.

File: integration-tests/README.md [hunk 2]
```diff
@@ -132,7 +132,7 @@ Other custom configuration settings available:
     is specified
   * `-D coreapi_server_image=bayesian/bayesian-api`
     (optional, default is `bayesian/bayesian-api`) - name of Bayesian core API server image
-  * `-D cucos_worker_image=bayesian/cucos-worker` (optional, default is `bayesian/cucos-worker`) - name of Bayesian
+  * `-D f8a_worker_image=bayesian/cucos-worker` (optional, default is `bayesian/cucos-worker`) - name of Bayesian
```
Comment (jpopelka) [inline integration-tests/README.md]: The [code](https://github.com/fabric8-analytics/fabric8-analytics-common/blob/master/integration-tests/features/environment.py#L264) actually reads from `coreapi_worker_image` configuration setting, so please rename to that one.

**Comment by fridex** [comment]
Thanks Jirka for the review!

I would like to get https://github.com/fabric8-analytics/fabric8-analytics-worker/pull/27 in before rebase.

**Comment by fridex** [comment]
Now I see that https://github.com/fabric8-analytics/fabric8-analytics-common/pull/16 is duplicate, but updated and rebased. Closing this in respect to #16.

**Comment by jpopelka** [comment]
> Now I see that #16 is duplicate, but updated and rebased. Closing this in respect to #16.

You mean #83 , don't you ?

**Comment by fridex** [comment]
> You mean #83 , don't you ?

Yes :)

**Pull request closed without merging**